mypy.ini

```diff
@@ -1,7 +1,7 @@
 [mypy]
 # CI should test for all versions, local development gets hints for oldest supported
-# Some upstream typeshed distutils stubs fixes are necessary before we can start testing on Python 3.12
-python_version = 3.8
+# But our testing setup doesn't allow passing CLI arguments, so local devs have to set this manually.
+# python_version = 3.8
```

Comment:
Would love if there was a way to tell the CI to pass the `--python-version` argument to override this config so that IDEs and maybe default local runs can test against the oldest supported Python version.

Reply:
`pytest-mypy` mentions the following:

> Mypy supports [reading configuration settings](http://mypy.readthedocs.io/en/latest/config_file.html) from a mypy.ini file. Alternatively, the plugin can be configured in a conftest.py to invoke mypy with extra options:
> 
> ```python
> def pytest_configure(config):
>     plugin = config.pluginmanager.getplugin('mypy')
>     plugin.mypy_argv.append('--check-untyped-defs')
> ```

Not sure if that is useful to achieve what you want via `conftest.py`.

Reply:
I think it would, something like:
```py
def pytest_configure(config):
    plugin = config.pluginmanager.getplugin('mypy')
    plugin.mypy_argv.append(f'--python-version={sys.version_info.major}.{sys.version_info.minor}')
```

Reply:
> Since you approved, mind if I tackle your comments in a follow-up ?

Sure!

----

> `plugin.mypy_argv.append(f'--python-version={sys.version_info.major}.{sys.version_info.minor}')`

Is that going to behave differently than just omitting `python_version` in `mypy.ini`?

Reply:
Honestly, I never run pytest-mypy locally. But yes that would mean that the mypy-test commands would always check for the installed python. A more complete snippet of code would probably check if this is being run on the CI and whether the dev already passed `--python-version` as to not overwrite it (let's say they run `pytest-mypy` directly w/o tox)